lib/statement.js

```diff
@@ -57,14 +59,49 @@ class Statement {
     });
   }
 
+  /**
+   * @description
+   * Associates parameter markers in an SQL statement to app variables
+   * @param {Array} params - the parameter list
+   * Each parameter element will also be an Array with 3 values (Value, In/out Type, Indicator)
+   * @returns {Promise} - Promise object represents the execution of bindParam()
+   * @memberof Statement
+   */
+  async bindParam(params) {
+    deprecate('As of 1.1.0, bindParam() is deprecated use bindParameters() instead.');
```

Comment:
After running the tests the deprecation warning will be:

```
Statement deprecated As of 1.1.0, bindParam() is deprecated use bindParameters() instead.
```

Perhaps we should reword this to:

```
Statement deprecated bindParam() as of 1.1.0. Use bindParameters() instead.
```

```suggestion
    deprecate('bindParam() as of 1.1.0. Use bindParameters() instead.');
```